Lib/Coproduct.hpp

```diff
@@ -33,12 +33,24 @@ namespace Lib {
 
 namespace TL = TypeList;
 
-template<unsigned v>
-struct Constant { static constexpr unsigned value = v; };
-
 template <class... As>
 class Coproduct;
 
+template<class Fs, class Ts>
```

Comment:
Could these have a one-line comment explaining what they do?

Reply:
it does "dead code" :)
Removed it!